src/components/Navbar.jsx

```diff
@@ -2,9 +2,11 @@ import { Routes, Route } from "react-router-dom";
 
 const Navbar = () => {
   return (
-    <Routes>
-      <nav className="Navbar">Navbar</nav>
-    </Routes>
+    <nav className="Navbar">
+      <span>Profile</span>
+      <span>Settings</span>
+      <span>Favorites</span>
+    </nav>
   );
```

Comment:
From a semantic perspective I would probably not use a span create the layout of the items in your nav bar. Instead I would look to use CSS to implement this. Take a look at flex as I think it'll be your friend here!